Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1000,12 +1005,17 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
                 }
                 return false;
             }
+
+            beginTrackingSelection(*pageIndex, pointInPageSpace, selectionGranularityForMouseEvent(event), event.modifiers()); // standard text selection
```

Comment:
Probably remove the comment

Reply:
Stray comment. Removing soon

Reply:
Removed in [53cae5f](https://github.com/WebKit/WebKit/commit/53cae5fe11eca85d9ea822cdec0181e4a4f4dcf6)